Add "More..." chapter (#2746)
github/opensource.guide: _articles/metrics.md

## Use 📊 to learn about people

Understanding metrics will help you build an active, growing open source project. Even if you don't track every metric on a dashboard, use the framework above to focus your attention on the type of behavior that will help your project thrive.

> 
> ## More...
> 
> There's a welcoming open source community around open source metrics, called [CHAOSS](https://chaoss.community/), in case you have doubts or want to dive deeper. They develop metrics and provide free software tools.

Reviewer: How about something like this:

```suggestion

[CHAOSS](https://chaoss.community/) is a welcoming, open source community focused on analytics, metrics and software for community health.
```

Reviewer 2: @misc{ wiki:xxx,
   author = "Meta",
   title = "Celebrate Women --- Meta{,} discussion about Wikimedia projects",
   year = "2022",
   url = "\url{https://meta.wikimedia.org/w/index.php?title=Celebrate_Women&oldid=22984427}",
   note = "[Online; accessed 19-March-2022]"
 }
